Formula/amber.rb

```diff
@@ -0,0 +1,18 @@
+class Amber < Formula
+  desc "CLI client for generating, scaffolding Amber web applications"
+  homepage "https://www.amberframework.org"
+  url "https://github.com/amberframework/amber/archive/v0.5.0.tar.gz"
```

Comment:
Looks like there's a newer version now 🙈.

Reply:
Yes, there is, and recently there was a new version 6.0 released. I was hoping the formula to be merge by now. :(

Reply:
Homebrew/core essentially has one regular maintainer, which would be @ilovezfs. It's fair to say it's a lot for one person to oversee.

Reply:
I understand being a maintainer of a popular project myself. 👍